contributors.ts

```diff
@@ -19,6 +19,10 @@ const contributors = {
     name: "רותם הורוביץ",
     linkedin: "https://www.linkedin.com/in/rotem-lurx-horovitz-9601705/",
   },
+  6: {
+    name: "הדס שווקי תמם",
```

Comment:
Hi! all looks good besides this - it should be:
הדס שוויקי-תימן

Reply:
Done 👍